runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -58,6 +58,8 @@
 	private static final int BORDER_RIGHT = 2;
 	private static final int BORDER_BOTTOM = 2;
 	private static final int PADDING = 2;
+	//As far as I can tell chatbox width cannot change, and this removes the need to create a widget to get the width.
+	private static final int CHATBOX_WIDTH = 519;
```

Comment:
It will still be a lot better to just get the chatbox bounds rather than having just constants lying around, because it might change in the future.